### PR TITLE
Leaderboard page

### PR DESCRIPTION
**Pull request by florisdobber**
Created a leader board page based on the Figma designs. Also added a picture to the assets as part of the new page. See screenshots below for result

leaderboard
progress

### CODE REVIEW

File: src/views/Leaderboard.vue [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        <div class="row">
+          <p class="rank">5</p>
+          <p>Team 5</p>
+          <p class="points">10</p>
+        </div>
+      </div>
+      <!-- My progress -->
+      <div class="progress" v-if="!showPlayers">
+        <h2>Team 3</h2>
+        <img src="../assets/tree 2.png" alt="Icon">
```
Comment (jackblanc) [inline src/views/Leaderboard.vue:45]: Let's avoid adding images into the assets folder. I'll write a ticket to make an S3 instance for LLB (and SFTT).

File: src/views/Leaderboard.vue [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    // show the teams progress
+    openProgress() {
+      this.showPlayers = false;
+    },
+  },
+};
+</script>
+
+<style scoped lang="less">
+.container {
```
Comment (jackblanc) [inline src/views/Leaderboard.vue:81]: Don't know how any of this works really, but looks good.

File: src/views/Leaderboard.vue [hunk 3]
```diff
@@ -0,0 +1,187 @@
+<template>
+  <div class="container">
+    <h1>Leaderboard</h1>
+    <div class="board">
+      <!-- heading to select view -->
+      <div class="heading">
+        <button class="active" @click="openPlayers" v-if="showPlayers">Players</button>
```
Comment (jackblanc) [inline src/views/Leaderboard.vue:7]: could we toggle the classname here instead of having two tags and the v-if?

File: src/views/Leaderboard.vue [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+        <button class="inactive" @click="openPlayers" v-if="!showPlayers">Players</button>
+        <div class=divider></div>
+        <button class="active" @click="openProgress" v-if="!showPlayers">My progress</button>
+        <button class="inactive" @click="openProgress" v-if="showPlayers">My progress</button>
+      </div>
+      <!-- Players -->
+      <div class="rows" v-if="showPlayers">
+        <!-- replace with v-for once data is loaded in -->
```
Comment (jackblanc) [inline src/views/Leaderboard.vue:15]: You should create a separate component for each of the Players and My Progress pages. Generally, we want to always break down the page into the smallest pieces we can.

File: src/views/Leaderboard.vue (hunk 1 above)
Comment (florisdobber) [reply src/views/Leaderboard.vue:45]: Removed from assets folder

**Comment by florisdobber** [comment]
- moved players and progress views out to components
- implemented a v-for in players based on dummy data
- slightly adjusted spacing on views
- removed previously added file from assets
- toggled class names for simplicity

**Pull request merged**